components/style/gecko_conversions.rs

```diff
@@ -0,0 +1,32 @@
+//! This module contains conversion helpers between Servo and Gecko types
+//! Ideally, it would be in geckolib itself, but coherence
+//! forces us to keep the traits and implementations here
```

Comment:
Question: all uses of these two `From` impls are going to be in the geckolib crate, so why can't they live in ports/geckolib/values.rs?